app/models/framework/definition/RM3756.rb

```diff
@@ -6,56 +6,107 @@ class RM3756 < Base
-        field 'Total Cost (ex VAT)', :decimal, exports_to: 'InvoiceValue'
-        field 'VAT Amount Charged', :decimal, exports_to: 'VATCharged'
+        field 'Tier Number', :string, exports_to: 'LotNumber', presence: true, inclusion: { in: %w[1 2] }
+        field 'Supplier Reference Number', :string, exports_to: 'SupplierReferenceNumber', presence: true
+        field 'Customer URN', :integer, exports_to: 'CustomerURN', urn: true
+        field 'Customer Organisation Name', :string, exports_to: 'CustomerName', presence: true
+        field 'Customer Post Code', :string, exports_to: 'CustomerPostCode', presence: true
+        field 'Customer Invoice Date', :string, exports_to: 'InvoiceDate', ingested_date: true
+        field 'Customer Invoice Number', :string, exports_to: 'InvoiceNumber', presence: true
+        field 'Service Type', :string, exports_to: 'ProductGroup', presence: true, case_insensitive_inclusion: { in: SERVICE_TYPE_VALUES }
+        field 'Primary Specialism', :string, exports_to: 'ProductDescription', presence: true, case_insensitive_inclusion: { in: PRIMARY_SPECIALISM_VALUES, message: 'must match the selected service type. For a list of service types and specialisms, check the lookups tab in the template.' }
+        field 'Practitioner Grade', :string, exports_to: 'ProductSubClass', presence: true, case_insensitive_inclusion: { in: PRACTITIONER_GRADE_VALUES }
+        field 'UNSPSC', :string, exports_to: 'UNSPSC', ingested_numericality: { only_integer: true }
+        field 'Unit of Purchase', :string, exports_to: 'UnitType', presence: true, case_insensitive_inclusion: { in: UNIT_OF_PURCHASE_VALUES }
+        field 'Price per Unit', :string, exports_to: 'UnitPrice', ingested_numericality: true
+        field 'Quantity', :string, exports_to: 'UnitQuantity', ingested_numericality: true
+        field 'Total Cost (ex VAT)', :string, exports_to: 'InvoiceValue', ingested_numericality: true
+        field 'VAT Amount Charged', :string, exports_to: 'VATCharged', ingested_numericality: true
         field 'Cost Centre', :string
         field 'Contract Number', :string
-        field 'Matter Name', :string, exports_to: 'CustomerReferenceNumber'
-        field 'Pricing Mechanism', :string, exports_to: 'Additional5'
-        field 'Award Procedure', :string, exports_to: 'Additional6'
-        field 'Pro-Bono Price per Unit', :decimal, exports_to: 'Additional1'
-        field 'Pro-Bono Quantity', :decimal, exports_to: 'Additional2'
-        field 'Pro-Bono Total Value', :decimal, exports_to: 'Additional3'
-        field 'Sub-Contractor Name (If Applicable)', :string, exports_to: 'Additional4'
+        field 'Matter Name', :string, exports_to: 'CustomerReferenceNumber', presence: true
+        field 'Pricing Mechanism', :string, exports_to: 'Additional5', presence: true, case_insensitive_inclusion: { in: PRICING_MECHANISM_VALUES }
+        # field 'Award Procedure', :string, exports_to: 'Additional6', presence: true
```

Comment:
Why did this get commented out?

Reply:
It exists in the MISO CSV as a field we need to export, however it doesn't exist in the template - so that presence check would always fail. I suspect the template was updated at some point and those changes were not reflected in MISO.